src/app/routes/assets/verification-method/verification-method.component.ts

```diff
@@ -61,6 +59,19 @@ export class VerificationMethodComponent implements OnInit {
     });
   }
 
+  getPublicKeyErrorMsg() {
+    debugger;
```

Comment:
?

Reply:
Yeah, forgot to remove that. Thanks for finding it out.